fasttrack/cli.py

```diff
@@ -34,12 +40,16 @@
 
 log = structlog.get_logger()
 
+DEFAULT_FASTTRACK_PORT = int(os.environ.get("FASTTRACK_PORT", 8082))
 CURRENT_DIR = Path(__file__).parent
 DAG_FASTTRACK_PATH = BASE_DIR / "dag_files/dag_fasttrack.yml"
 DUMMY_DATA = {
     "sheet_url": "https://docs.google.com/spreadsheets/d/e/2PACX-1vS3kx1WYuCWfWVQYuu3H0J0E2epv3jX9-odkzvhuHj-XenGGFvTCKIOngNpbkxkmw8iv4ZS5eNF4hS0/pub?output=csv",
```

Comment:
If this is going to stay as an example, shouldn't it be a real dummy example, instead of excess mortality data?

Reply:
Removed it altogether, it's not as useful as in walkthrough.